galaxy_release_util/point_release.py

```diff
@@ -612,7 +618,20 @@ def build_and_upload(
         packages.append(package)
     for package in packages:
         click.echo(f"Building package {package}")
+        if package.name == "meta":
+            build_meta_dependencies(package, packages, get_root_version(galaxy_root))
         build_package(package)
+    return packages
+
+
+@group_options(galaxy_root_option, packages_option, no_confirm_option)
+@cli.command("build-and-upload", help="Build and upload packages")
+def build_and_upload(
+    galaxy_root: Path,
+    package_subset: List[str],
+    no_confirm: bool,
+):
+    packages: List[Package] = _build(galaxy_root, package_subset)
```

Comment:
```suggestion
    packages = _build(galaxy_root, package_subset)
```

Annotation should come from `_build()` return type.